From 045319f1a8b855dee0a829b0e82548a59d32aa05 Mon Sep 17 00:00:00 2001
From: Hyunjung Kom <hjnim.kim@samsung.com>
Date: Tue, 24 Jul 2012 21:56:00 +0900
Subject: [PATCH] JQM: Add ui-btn-box-theme class to real buttons

Signed-off-by: Hyunjung Kim <hjnim.kim@samsung.com>
---
 .../js/jquery.mobile.buttonMarkup.js               |    5 +++++
 1 file changed, 5 insertions(+)

diff --git a/libs/js/jquery-mobile-1.1.0/js/jquery.mobile.buttonMarkup.js b/libs/js/jquery-mobile-1.1.0/js/jquery.mobile.buttonMarkup.js
index d1b992b..d523ad9 100644
--- a/libs/js/jquery-mobile-1.1.0/js/jquery.mobile.buttonMarkup.js
+++ b/libs/js/jquery-mobile-1.1.0/js/jquery.mobile.buttonMarkup.js
@@ -93,6 +93,11 @@ $.fn.buttonMarkup = function( options ) {
 		buttonClass += o.shadow ? " ui-shadow" : "";
 		buttonClass += o.corners ? " ui-btn-corner-all" : "";
 
+		// To distinguish real buttons
+		if( el.jqmData("role") == "button" || e.tagName == "BUTTON" ){
+			buttonClass += " ui-btn-box-" + o.theme;
+		}
+
 		/* TIZEN style markup */
 		buttonStyle = el.jqmData("style");
 
-- 
1.7.9.5

