From a0e5981b139c44c505acb6d07455b82deccc2c8e Mon Sep 17 00:00:00 2001
From: Minkyu Kang <mk7.kang@samsung.com>
Date: Fri, 31 May 2012 09:18:16 +0900
Subject: [PATCH] JQM: add tizen non-animation transition handler

Signed-off-by: Minkyu Kang <mk7.kang@samsung.com>
---
 .../js/jquery.mobile.navigation.js                 |   19 +++++++++++++++++++
 1 file changed, 19 insertions(+)

diff --git a/libs/js/jquery-mobile-1.0.1pre/js/jquery.mobile.navigation.js b/libs/js/jquery-mobile-1.0.1pre/js/jquery.mobile.navigation.js
index 1bf187f..aa33160 100755
--- a/libs/js/jquery-mobile-1.0.1pre/js/jquery.mobile.navigation.js
+++ b/libs/js/jquery-mobile-1.0.1pre/js/jquery.mobile.navigation.js
@@ -630,6 +630,25 @@
 		return $.Deferred().resolve( name, reverse, $toPage, $fromPage ).promise();
 	};
 
+	//tizen non-animation transition handler
+	$.mobile.tizenTransitionHandler = function( name, reverse, $toPage, $fromPage ) {
+		var deferred = new $.Deferred(),
+		    doneFunc = function () {
+			if ( $fromPage ) {
+				$fromPage.removeClass( $.mobile.activePageClass );
+			}
+			$toPage.addClass( $.mobile.activePageClass );
+
+			deferred.resolve( name, reverse, $toPage, $fromPage );
+		    };
+
+		setTimeout( doneFunc, 0);
+
+		return deferred.promise();
+	};
+
+	$.mobile.noneTransitionHandler = $.mobile.tizenTransitionHandler;
+
 	//default handler for unknown transitions
 	$.mobile.defaultTransitionHandler = $.mobile.noneTransitionHandler;
 
-- 
1.7.9.5

